assets/styles/informed.scss

```diff
@@ -2044,3 +2044,35 @@ caption {
 .session-expiry-popup__countdown {
   margin-top: 1.3em;
 }
+
+.upload-progress--outline {
+  border: 1px solid #1d70b8;
+  height: 30px;
+  position: absolute;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.upload-progress--bar {
```

Comment:
for BEM I would probably expect these to be elements. 

block: `.upload-progress` which encapsulates all of these styles
elements:
- `.upload-progress__outline` 
- `.upload-progress__bar`

modifier:
- `.upload-progress__bar--stripes` 

generally if you have modifiers before elements, eg `.upload-progress--bar__stripes` then it's a sign that things need to be split up differently.


also, `.upload-progress__outline` seems to be an empty div. could it be something like this
```
.upload-progress__bar--outline {
  &:before(or after) {
  ...
  }
}
```
and use it like so
```
<div class="js-upload-progress-bar upload-progress__bar upload-progress__bar--outline" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100">
```

Reply:
Hmm, I understand the renaming but I don't understand the benefit of the `upload-progress__bar ` class since it doesn't contain any styles. Is it purely for convention?

Reply:
not sure I follow. your .upload-progress--bar has styles, but it should be named ` .upload-progress__bar`

Reply:
Sorry, I read the code wrong I saw:
```html
<div class="js-upload-progress-bar upload-progress__bar upload-progress__bar--outline"
```
as
```html
<div class="js-upload-progress-bar upload-progress upload-progress__bar"
```
🤦 
Still slowly getting into things after the Christmas break :)

Reply:
Ha, the only times I've ever used the pseudo-classes `:before` or `:after` is when I've copied css from someone and need to edit it, never actually looked into how they work. Thanks for the tip 👍